tests/StackExchange.Redis.Tests/SortedSets.cs

```diff
@@ -52,6 +61,153 @@ public class SortedSets : TestBase
+
+                var inter = db.SortedSetCombine(SetOperation.Intersect, new RedisKey[]{ key1, key2});
+                Assert.Equal(5, inter.Length);
+                Assert.Equal("a", inter[0]);
+
+                var union = db.SortedSetCombine(SetOperation.Union, new RedisKey[]{ key1, key2});
+                Assert.Equal(10, union.Length);
+                Assert.Equal("a", union[0]);
+            }
+        }
+
+        [Fact]
+        public void SortedSetCombineWithScores()
+        {
+            using (var conn = Create())
+            {
+                Skip.IfBelow(conn, RedisFeatures.v6_2_0);
+
+                var db = conn.GetDatabase();
+                var key1 = Me();
+                db.KeyDelete(key1, CommandFlags.FireAndForget);
+                var key2 = Me() + "2";
+                db.KeyDelete(key2, CommandFlags.FireAndForget);
+
+                db.SortedSetAdd(key1, entries);
+                db.SortedSetAdd(key2, entriesPow3);
+
+                var diff = db.SortedSetCombineWithScores(SetOperation.Difference, new RedisKey[]{ key1, key2});
+                Assert.Equal(5, diff.Length);
+                Assert.Equal(new SortedSetEntry("b", 2), diff[0]);
+
+                var inter = db.SortedSetCombineWithScores(SetOperation.Intersect, new RedisKey[]{ key1, key2});
+                Assert.Equal(5, inter.Length);
+                Assert.Equal(new SortedSetEntry("a", 2), inter[0]);
+
+                var union = db.SortedSetCombineWithScores(SetOperation.Union, new RedisKey[]{ key1, key2});
+                Assert.Equal(10, union.Length);
+                Assert.Equal(new SortedSetEntry("a", 2), union[0]);
+            }
+        }
+
+        [Fact]
+        public void SortedSetCombineAndStore()
+        {
+            using (var conn = Create())
+            {
+                Skip.IfBelow(conn, RedisFeatures.v6_2_0);
+
+                var db = conn.GetDatabase();
+                var key1 = Me();
+                db.KeyDelete(key1, CommandFlags.FireAndForget);
+                var key2 = Me() + "2";
+                db.KeyDelete(key2, CommandFlags.FireAndForget);
+                var destination = Me() + "dest";
+                db.KeyDelete(destination, CommandFlags.FireAndForget);
+
+                db.SortedSetAdd(key1, entries);
+                db.SortedSetAdd(key2, entriesPow3);
+
+                var diff = db.SortedSetCombineAndStore(SetOperation.Difference, destination, new RedisKey[]{ key1, key2});
+                Assert.Equal(5, diff);
+
+                var inter = db.SortedSetCombineAndStore(SetOperation.Intersect, destination, new RedisKey[]{ key1, key2});
+                Assert.Equal(5, inter);
+
+                var union = db.SortedSetCombineAndStore(SetOperation.Union, destination, new RedisKey[]{ key1, key2});
+                Assert.Equal(10, union);
+            }
+        }
+
+        [Fact]
+        public void SortedSetCombineErrors()
+        {
+            using (var conn = Create())
```

Comment:
On all new tests that are full scope, let's prefer the new `using` format please! (I adjusted other PRs, happy to push here too lemme know)

Reply:
done :) thank you